src/Entities/Pickup.cpp

```diff
@@ -159,7 +165,7 @@ void cPickup::Tick(std::chrono::milliseconds a_Dt, cChunk & a_Chunk)
 			}
 
 			// Try to combine the pickup with adjacent same-item pickups:
-			if ((m_Item.m_ItemCount < m_Item.GetMaxStackSize()))  // Don't combine if already full
+			if ((m_Item.m_ItemCount < m_Item.GetMaxStackSize()) && this->IsOnGround())  // Don't combine if already full or not on ground
```

Comment:
There's no problem, it's just unnecessary, and slightly distracting ("why is it there and no-where else? What did the dev want to say by that?")